modules/migration-upgrading-mtc-on-ocp-4.adoc

```diff
@@ -0,0 +1,59 @@
+// Module included in the following assemblies:
+// * migration/migrating_3_4/upgrading-migration-tool-3-4.adoc
+// * migration/migrating_4_1_4/upgrading-migration-tool-4-1-4.adoc
+// * migration/migrating_4_2_4/upgrading-migration-tool-4-2-4.adoc
+
+[id='migration-upgrading-mtc-on-ocp-4_{context}']
+= Upgrading the {mtc-full} on an {product-title} 4 cluster
+
+You can upgrade the {mtc-full} ({mtc-short}) on an {product-title} 4 cluster by using the {product-title} web console.
+
+.Prerequisites
+
+* You must be logged in as a user with `cluster-admin` privileges.
+
+.Procedure
+
+. In the {product-title} console, navigate to *Operators* -> *Installed Operators*.
++
+Operators that have a pending upgrade display an *Upgrade available* status.
+
+. Click *{mtc-full} Operator*.
+. Click the *Subscription* tab. Any upgrades requiring approval are displayed next to *Upgrade Status*. For example, it might display *1 requires approval*.
+. Click *1 requires approval*, then click *Preview Install Plan*.
+. Review the resources that are listed as available for upgrade and click *Approve*.
+. Navigate back to the *Operators -> Installed Operators* page to monitor the progress of the upgrade. When complete, the status changes to *Succeeded* and *Up to date*.
+ifdef::migrating-4-1-4,migrating-4-2-4[]
+. Click *{mtc-full} Operator*.
+. Under *Provided APIs*, locate the *Migration Controller* tile, and click *Create Instance*.
+. If you are upgrading {mtc-short} on a _source_ cluster, update the following parameters in the `MigrationController` custom resource (CR) manifest:
+endif::[]
+ifdef::migrating-4-1-4[]
++
+[source,yaml]
+----
+spec:
+...
+  migration_controller: false
+  migration_ui: false
+...
+  deprecated_cors_configuration: true
+----
+endif::[]
+ifdef::migrating-4-2-4[]
++
+[source,yaml]
+----
+spec:
+...
+  migration_controller: false
+  migration_ui: false
+----
+endif::[]
+ifdef::migrating-4-1-4,migrating-4-2-4[]
++
+You do not need to update the `MigrationController` CR manifest on the target cluster.
+
+. Click *Create*.
+endif::[]
+. Click *Workloads* -> *Pods* to verify that the {mtc-short} pods are running.
```

Comment:
Do we want a step here to make sure users who need to do this procedure are aware it is there? Something like:

. If you are upgrading the Migration Toolkit for Containers (MTC) version 1.3.x to 1.4, follow the steps in _Upgrading MTC 1.3 to 1.4_ in the following section.

Reply:
It is problematic in modular docs to refer to other modules.